ColabTool/Shared/Post/story.jsx

```diff
@@ -13,7 +13,7 @@ const draftDetails = {
   createdAt: 'on March 2nd at 12:45pm (GMT)',
   email: 'ash@buffer.com',
   isRetweet: false,
-  postAction: 'This post is scheduled for 9:42pm (GMT)',
+  postAction: 'This post will be added to the queue',
```

Comment:
Might be nice to create some demo post data and import it in these story files. That way we only need to change this in one place. (A task for a later followup commit)